ChaosMod/Effects/db/Player/GolfPercent.cpp

```diff
@@ -0,0 +1,84 @@
+/*
+	Effect by Vladimir Gotsin
+*/
+
+#include <stdafx.h>
+
+static void OnStart()
+{
+	//vehicle
+	auto playerPed = PLAYER_PED_ID();
+	float playerHeading = GET_ENTITY_HEADING(playerPed);
+	float heading = GET_ENTITY_HEADING(IS_PED_IN_ANY_VEHICLE(playerPed, false) ? GET_VEHICLE_PED_IS_IN(playerPed, false) : playerPed);
+
+	Vector3 playerPos = GET_ENTITY_COORDS(playerPed, false);
+
+	static const Hash caddyModel = GET_HASH_KEY("CADDY");
+
+	Vehicle veh = CREATE_VEHICLE(caddyModel, playerPos.x, playerPos.y, playerPos.z, heading, true, true, true);
```

Comment:
Use `CreateTempVehicle` here instead as otherwise it will stay on the map forever and won't despawn.